package.json

```diff
@@ -12,37 +12,36 @@
     "lint-staged": "lint-staged"
   },
   "dependencies": {
-    "@emotion/react": "^11.11.4",
-    "@emotion/styled": "^11.11.5",
-    "@mui/icons-material": "^5.15.21",
-    "@mui/material": "^5.15.21",
-    "@mui/x-data-grid": "^7.8.0",
+    "@emotion/react": "^11.13.3",
+    "@emotion/styled": "^11.13.0",
+    "@mui/icons-material": "^6.1.5",
+    "@mui/material": "^6.1.5",
+    "@mui/x-data-grid": "^7.22.0",
     "csv-parse": "^5.5.6",
     "immer": "^10.1.1",
-    "next": "14.2.4",
+    "next": "15.0.2",
     "react": "18.3.1",
     "react-dom": "18.3.1",
-    "react-hook-form": "^7.52.0",
+    "react-hook-form": "^7.53.1",
     "react-to-print": "^2.15.1",
```

Comment:
encountered a bug `"Printing failed because the `contentWindow` of the print iframe did not load. This is possibly an error with `react-to-print`. Please file an issue: https://github.com/MatthewHerbst/react-to-print/issues/"` in `react-to-print` v3.0.2. I couldn't print, so deferred to update